src/parsers/manifestjson.js

```diff
@@ -314,6 +314,15 @@ export default class ManifestJSONParser extends JSONParser {
           'page'
         );
       }
+      if (this.parsedJSON.background.service_worker) {
+        // TODO: replace this with a call to this.validateFileExistsInPackage
```

Comment:
Any reference to an issue?

Reply:
sure, I was going to file the follow up issue and mention it in this comment after double-checking that we agreed on doing this in this PR.